langchain4j-milvus/src/main/java/dev/langchain4j/store/embedding/milvus/MilvusEmbeddingStore.java

```diff
@@ -25,21 +25,25 @@
 import io.milvus.param.dml.InsertParam;
 import io.milvus.param.dml.SearchParam;
 import io.milvus.response.SearchResultsWrapper;
+import org.apache.commons.lang3.StringUtils;
+
 import java.util.ArrayList;
+import java.util.Collections;
 import java.util.List;
 
 /**
  * Represents an <a href="https://milvus.io/">Milvus</a> index as an embedding store.
  * Does not support storing {@link dev.langchain4j.data.document.Metadata} yet.
  */
-public class MilvusEmbeddingStore implements EmbeddingStore<TextSegment> {
+public class MilvusEmbeddingStore implements MilvusEmbeddingStoreExtend<TextSegment> {
 
   static final String ID_FIELD_NAME = "id";
   static final String TEXT_FIELD_NAME = "text";
   static final String VECTOR_FIELD_NAME = "vector";
 
   private final MilvusServiceClient milvusClient;
   private final String collectionName;
+  private final String partitionName;
```

Comment:
I am not sure we should include `partitionName` as a parameter of `MilvusEmbeddingStore`.
As far as I understand, `partitionName` is accepted by the constructor with a single purpose: to create a partition. `partitionName` is not used later on in the lifecycle of `MilvusEmbeddingStore`. I suppose it will be more intuitive to provide a method `public void createPartition(StringPartitionName)` instead.